### PR TITLE
[FR] `about` folder translated in French

### PR DESCRIPTION
**Pull request by SamuelAlev**

### CODE REVIEW

**Comment by SamuelAlev** [comment]
Thank you, I will know for the next ones.

**Comment by lamberta** [comment]
Thanks. Will give this pull request some time for a reviewer to look at.

I'd recommend starting with the [quickstart tutorial](https://github.com/tensorflow/docs/tree/master/site/en/tutorials/quickstart) notebooks and then the [Keras tutorials](https://github.com/tensorflow/docs/tree/master/site/en/tutorials/keras). And then just work your way down the [tutorials left nav](https://www.tensorflow.org/tutorials) or whatever you find interesting. These are the more popular sections of the docs.

For notebook workflows, see the [TF docs contributor guide](https://www.tensorflow.org/community/contribute/docs) and the [notebook template](https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb). Testing in Colab is usually the easiest way.

Thanks for contributing!

File: site/fr/about/bib.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+
+### Au format BibTeX
+
+Si vous utilisez TensorFlow dans votre recherche et souhaitez citer le système TensorFlow, nous vous suggérons de citer ce livre blanc.
```
Comment (ArmanSoltani) [inline site/fr/about/bib.md]:
```suggestion
Si vous utilisez TensorFlow dans vos recherches et souhaitez citer le système TensorFlow, nous vous suggérons de citer ce livre blanc.
```

File: site/fr/about/bib.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+2015. Software available from tensorflow.org.
+</pre>
+
+
+
+## TensorFlow: un système pour l'apprentissage automatique à grande échelle
+
+[Accéder à ce livre blanc.](https://www.usenix.org/system/files/conference/osdi16/osdi16-abadi.pdf)
+
+**Résumé:** TensorFlow est un système d'apprentissage automatique qui fonctionne à grande échelle et dans des environnements hétérogènes. TensorFlow utilise des graphiques de flux de données pour représenter le calcul, l'état partagé et les opérations qui font muter cet état.
```
Comment (ArmanSoltani) [inline site/fr/about/bib.md]:
```suggestion
**Résumé:** TensorFlow est un système d'apprentissage automatique qui fonctionne à grande échelle et dans des environnements hétérogènes. TensorFlow utilise des graphes de flux de données pour représenter le calcul, l'état partagé et les opérations qui font muter cet état.
```

File: site/fr/about/bib.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+Il mappe les nœuds d'un graphe de flux de données sur de nombreuses machines d'un cluster, et au sein d'une machine sur plusieurs périphériques de calcul, y compris les processeurs multicœurs, les GPU polyvalents et les ASIC personnalisés appelés Tensor Processing Units (TPU).
+Cette architecture donne de la flexibilité au développeur d'applications : alors que dans les précédentes conceptions “serveur de paramètres”, la gestion des états partagés est intégrée au système, TensorFlow permet aux développeurs d'expérimenter de nouvelles optimisations et algorithmes d'apprentissage.
+TensorFlow supporte une variété d'applications, avec un accent sur la formation et l'inférence des réseaux d'apprentissage profond.
```
Comment (ArmanSoltani) [inline site/fr/about/bib.md]:
```suggestion
TensorFlow supporte un grand nombre d'applications, avec un accent mis sur l'entraînement et l'inférence des réseaux d'apprentissage profond.
```

File: site/fr/about/bib.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+Plusieurs services Google utilisent TensorFlow en production, nous l'avons publié en tant que projet open-source, et il est devenu largement utilisé pour la recherche en apprentissage automatique.
+Dans ce livre, nous décrivons le modèle de flux de données de TensorFlow et démontrons les performances convaincantes que TensorFlow peut atteindre pour plusieurs applications du monde réel.
```
Comment (ArmanSoltani) [inline site/fr/about/bib.md]:
```suggestion
Dans ce livre blanc, nous décrivons le modèle de flux de données de TensorFlow et démontrons les performances intéressantes que TensorFlow permet d'atteindre dans plusieurs applications du monde réel.
```

**Pull request merged**